tests/unittests/ThreadPoolExecutorTest.cpp

```diff
@@ -93,7 +97,13 @@ class TestDeviceManager final : public runtime::DeviceManager {
 
         for (auto p : registeredResult->resultContext->getPlaceholderBindings()
                           ->pairs()) {
-          context->getPlaceholderBindings()->get(p.first)->assign(p.second);
+          Tensor *T = context->getPlaceholderBindings()->get(p.first);
+          if (T) {
+            T->assign(p.second);
+          } else {
+            context->getPlaceholderBindings()->insert(p.first,
```

Comment:
When is this branch taken? I was under the impression that all `Placeholders` used by a function have to be mapped in the `PlaceholderBindings` that it is executed with.

Reply:
You are right here, it's the input/result context that don't need to have all Placeholders in them.